### PR TITLE
Added  the Downloads folder

### PR DESCRIPTION
**Pull request by cjprogamer**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -226,4 +224,13 @@ tags
 !.vscode/extensions.json
 .history
 
+<<<<<<< HEAD
```
Comment (kmehant) [inline .gitignore]: Fix these conflicts

File: .gitignore [hunk 2]
```diff
@@ -226,4 +224,13 @@ tags
...
+###make empty downloads folder
+downloads/*
+!downloads/.gitkeep 
+=======
+downloads/*
+!downloads/.gitkeep
```
Comment (kmehant) [inline .gitignore]: Can you explain the reason .`gitkeep` is used here 
Can I use something like `.downloads` or something else?
Put forward your explanation.

**Comment by kmehant** [comment]
Squash your commits

**Comment by kmehant** [comment]
Work on `develop` branch not on master

File: .gitignore (hunk 2 above)
Comment (cjprogamer) [reply .gitignore]: Is it correct if I add these lines ? 
`#downloads
.downloads`

Comment (kmehant) [reply .gitignore]: That results In a comment due to #
> Can you explain the reason .gitkeep is used here
Can I use something like .downloads or something else?

Hopefully you will answer this

Comment (cjprogamer) [reply .gitignore]: no, I mean,
" #downlaods
    .downloads

Comment (cjprogamer) [reply .gitignore]: > That results In a comment due to #
> 
> > Can you explain the reason .gitkeep is used here
> > Can I use something like .downloads or something else?
> 
> Hopefully you will answer this

I thought that code would not delete the downloads folder

**Comment by kmehant** [comment]
#843 shifted

**Pull request closed without merging**